ecf/cam/prep/jevs_hireswarw_grid2obs_prep.ecf

```diff
@@ -78,7 +78,6 @@ export MET_CONFIG="${MET_PLUS_PATH}/parm/met_config"
 export PYTHONPATH=$HOMEevs/ush/$COMPONENT:$PYTHONPATH
 
 # Developer Settings
-export envir=prod
 export DATA=/lfs/h2/emc/stmp/$USER/evs_test/$envir/tmp
```

Comment:
@MarcelCaron-NOAA  removing the definition of envir will break the definition of DATA in the next line. Should DATA be corrected?

Reply:
Oh! Nevermind ... I see that it is now exported from `run.ver`.

Reply:
Yes!  "envir" was added and then later removed from the ecf script in response to upstream changes in run.ver